### ISSUE TITLE
Config.unobserve is deprecated.

### ISSUE THREAD

**Issue by KoganezawaRyouta** [AtomLinter/linter-ruby#16]
Config::unobserve no longer does anything. Call `.dispose()` on the object returned by Config::observe instead.
```
Config.unobserve (/Applications/Atom.app/Contents/Resources/app.asar/src/config.js:1060:19)
LinterRuby.destroy (/Users/ryoutakoganezawa/.atom/packages/linter-ruby/lib/linter-ruby.coffee:30:16)
LinterView.remove (/Users/ryoutakoganezawa/.atom/packages/linter/lib/linter-view.coffee:252:6)
<unknown> (/Users/ryoutakoganezawa/.atom/packages/linter/lib/linter-view.coffee:124:7)
```

**Comment by envygeeks** [comment]
@KoganezawaRyouta Please update your linter-ruby this was fixed a few days ago.

**Comment by kevinsawicki** [comment]
Fixed by #15

**Comment by Arcanemagus** [comment]
Duplicate of #8.
